File: pkg/wow/raid.go
package wow

type (
	RaidRank int
	RaidRole string
)

const (
	RaidRankMember    RaidRank = 0
	RaidRankAssistant RaidRank = 1
	RaidRankLeader    RaidRank = 2
)

func GetNumGroupMembers() int {
	return 0
}

func GetRaidRosterInfo(idx int) (name string, rank RaidRank, subgroup, level int, class, fileName, zone string, online, isDead bool, role RaidRole, masterLooter bool) {
	return "", 0, 0, 0, "", "", "", false, false, "", false
}

File: pkg/wow/raid.lua
-- Package declaration
local _wow = _G["github.com/eandre/lunar-wow/pkg/wow"] or {}
_G["github.com/eandre/lunar-wow/pkg/wow"] = _wow

_wow.RaidRankMember = 0
_wow.RaidRankAssistant = 1
_wow.RaidRankLeader = 2

_wow.GetNumGroupMembers = function()
    return GetNumGroupMembers()
end

_wow.GetRaidRosterInfo = function(idx)
	local name, rank, subgroup, level, class, fileName, zone, online, isDead, role, isML = GetRaidRosterInfo(idx)
	online = online == 1
	isDead = isDead == 1
	isML = isML == 1
	return name, rank, subgroup, level, class, fileName, zone, online, isDead, role, isML
end
